crates/host/src/wasmbus/mod.rs

```diff
@@ -1933,7 +1933,7 @@ impl Host {
                 let mut queue = Abortable::new(queue, queue_abort_reg);
                 queue
                     .by_ref()
-                    .for_each({
+                    .for_each_concurrent(0, {
```

Comment:
Nit: having this parameter be 0 is a little confusing if you go look at the docs because it seems like you're telling it not to have _any_ concurrency. Consider setting this to `None` instead